CMakeLists.txt

```diff
@@ -52,8 +52,8 @@ find_package(sonata QUIET REQUIRED)
 
 include(PackageConfig)
 
-FILE(GLOB MVDTOOL_HEADERS "include/mvd/*.hpp")
-FILE(GLOB MVDTOOLE_BITS_HEADERS "include/mvd/*/*.hpp")
+FILE(GLOB MVDTOOL_HEADERS "include/mvd/*.hpp include/tsv/*.hpp")
```

Comment:
I think it is a good convention to install all headers in the same place for a given project.
You can still create the tsv dir inside if you really want to separate. @matz-e maybe we could rename `mvd` to `mvdtool`?

Reply:
If `mvd` folder is renamed to `mvdtool` then it will make sense to me to move the `tsv` related files inside. Otherwise I think that it is nice to have a distinction.

Reply:
I see you moved headers to mvdtool. Great! For me that's the beast approach